package.json

```diff
@@ -79,6 +79,7 @@
     "js-yaml": "^4.1.0",
     "jsdom": "^25.0.1",
     "smee-client": "^2.0.3",
-    "vitest": "^2.1.1"
+    "vitest": "^2.1.1",
+    "shortest": "workspace:*"
```

Comment:
What's this do?

Reply:
So I setup pnpm-workspace.yaml that allows for a better monorepo development of the package. This way we can build and install the package locally as if we have installed it from npm. I think it's good for testing the package during development.